pandas/io/excel.py

```diff
@@ -85,20 +87,36 @@
     Column (0-indexed) to use as the row labels of the DataFrame.
     Pass None if there is no such column.  If a list is passed,
     those columns will be combined into a ``MultiIndex``.  If a
-    subset of data is selected with ``usecols``, index_col
+    subset of data is selected with ``usecols_excel``, index_col
     is based on the subset.
 parse_cols : int or list, default None
 
     .. deprecated:: 0.21.0
        Pass in `usecols` instead.
 
-usecols : int or list, default None
+usecols : list-like or callable, default None
```

Comment:
was a single int allowed before?

Reply:
Yes, it was. Look: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_excel.html